src/Dapr.Actors/Runtime/ActorRuntimeOptions.cs

```diff
@@ -3,29 +3,117 @@
 // Licensed under the MIT License.
 // ------------------------------------------------------------
 
+using System;
```

Comment:
nit: inside namespace

Reply:
I'm going to log an issue to change this convention - it's not the default that any of the tools use.

Reply:
https://github.com/dapr/dotnet-sdk/issues/489